aep/general/0157/aep.md.j2

```diff
@@ -37,15 +40,41 @@ Field masks **should not** be specified in the
   [here](https://github.com/protocolbuffers/protobuf/issues/8547#issuecomment-2005180068).
   Consider using the view enumeration pattern described below instead instead.
 
-**Note:** Changing the default value of the field mask parameter is a
-[breaking change](./backwards-compatibility#semantic-changes).
+{% tab oas %}
+
+**Note:** OAS example not yet written.
+
+{% endtabs %}
 
 ### View enumeration
 
 Alternatively, an API **may** support partial responses with view enums. View
 enums are useful for situations where an API only wants to expose a small
 number of permutations to the user:
 
+- The enum **should** be specified as a `view` field on the request message.
+- The enum **should** be named something ending in `-View`
+- The enum **should** at minimum have values named `BASIC` and `FULL` (although
+  it **may** have values other than these).
+- The `UNSPECIFIED` value **must** be valid (not an error), and the API
+  **must** document what the unspecified value will do.
+  - For List methods, the effective default value **should** be `BASIC`.
+  - For Get methods, the effective default value **should** be either `BASIC`
+    or `FULL`.
+- APIs **may** add fields to a given view over time. APIs **must not** remove a
+  field from a given view (this is a breaking change).
+
+  **Note:** If a service requires (or might require) multiple views with
+  overlapping but distinct values, there is a potential for a namespace
```

Comment:
any reason why we shouldn't always just nest the enum under the individual resource? It seems like if the guidance is "only do it if you *think* there's going to be a conflict", that'll lead to a lot of unintended conflicts afterward.

Reply:
Yeah I agree, we already nest it at Roblox.  I just missed this when reviewing the copy/pasted content.  Updated.

Reply:
It looks like the original comment is here still saying if a service may require multiple views - can that be removed with just "always put a view under the resource"?

Reply:
Oh, whoops, removed.